static/app/components/commandPalette/context.tsx

```diff
@@ -0,0 +1,104 @@
+import {createContext, useCallback, useContext, useMemo, useReducer} from 'react';
+
+import type {CommandPaletteAction} from './types';
+
+type CommandPaletteProviderProps = {children: React.ReactNode};
+
+type CommandPaletteStore = {
```

Comment:
I saw you used the `store` naming, which in the context of sentry, might lead people to believe that this is a redux store, when really, it's just state. Wdyt about renaming this to use `CommandPaletteState` (with that also the file names)